tasks/collect-data/collect-data.yaml

```diff
@@ -201,3 +204,45 @@ spec:
             "$(workspaces.data.path)/$(params.subdirectory)/release_plan_admission.json"
 
         exit $RC
+    - name: print-pipeline-ref-info
+      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      script: |
+        #!/usr/bin/env bash
+        set +x
```

Comment:
What do you think about using `set -eux`? I checked and `-u` works fine with stuff like `org="${org:-unknown}"` even if it was undefined.

Reply:
I wanted to reduce the verbosity...so I am keeping `set +x` but I added `set -eu`. works fine

Reply:
As I explained in the other thread, this `set +x` is not needed. It's the default. You're seeing tracing in tests, because the mock injects `set -eux`. You could change the mock to do `set -eu`, but it's probably fine/better to have tracing when running tests.

Reply:
ahhhhhh! now I understand!

thanks. fixed